Scripts/validate-3ds2-SDK-version.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
-cardDetails3DS2SDKVersion=$(sed -En 's/^.*public.*let.*threeDS2SdkVersion.*:.*String.*=.*"([0-9]+.[0-9].[0-9]+)".*$/\1/p' AdyenCard/Components/Card/ThreeDS2SdkVersion.swift)
-cocoapods3DS2SDKVersion=$(sed -En "s/^.*dependency.*\'Adyen3DS2'.*,.*'([0-9]+.[0-9].[0-9]+)'.*$/\1/p" Adyen.podspec)
-carthage3DS2SDKVersion=$(sed -En 's/^.*github.*"adyen\/adyen-3ds2-ios".*==.*([0-9]+.[0-9].[0-9]+).*$/\1/p' Cartfile)
+cardDetails3DS2SDKVersion=$(sed -En 's/^.*public.*let.*threeDS2SdkVersion.*:.*String.*=.*"([0-9]+.[0-9]+.[0-9]+)".*$/\1/p' AdyenCard/Components/Card/ThreeDS2SdkVersion.swift)
+cocoapods3DS2SDKVersion=$(sed -En "s/^.*dependency.*\'Adyen3DS2'.*,.*'([0-9]+.[0-9]+.[0-9]+)'.*$/\1/p" Adyen.podspec)
```

Comment:
@mohammedDehairy is there a reason only to allow 0-9 as a valid value for minor versions of the 3DS2 SDK?

Reply:
no, i think it was a mistake, nice catch! :-)